src/Symfony/Component/Console/Descriptor/ReStructuredTextDescriptor.php

```diff
@@ -0,0 +1,291 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Console\Descriptor;
+
+use Exception;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\Helper;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputDefinition;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ReStructuredTextDescriptor extends Descriptor
+{
+    // <h1>
+    private string $partChar = '#';
```

Comment:
Let's be consistent with what we are using in the docs:

h1 =====
h2 -----
h3 ~~~~~
h4 .....